system/Build.jl

```diff
@@ -22,3 +23,11 @@ PackageCompiler.create_sysimage([:Oscar], sysimage_path=sysimage, precompile_exe
 
 println("(re)start julia as")
 println("\tjulia -J $(sysimage)")
+
+# Remove the packages we added, so that 
+# Aqua doesn't complain in future testing
```

Comment:
Can't we create a temporary environment, install packages just in there, and discard it later? That seems cleaner and simpler to me, and also reduces the risk of leaving things in a partially messed up state in case the user ctrl-C's at a bad moment.

Reply:
I am unable to make this idea work, and I'm not sure why it doesn't work.

As of commit `3b1f1abb` , trying to `Oscar.build()` fails, with one of the problems being that `Oscar.system("precompile.jl")` includes `precompile.jl` from the "system" installation of Oscar, not the the place pointed to by the curent `Project.toml`, but I don't understand *why* it chooses the wrong path.

(There are also other problems relating to missing source files of the extra packages I'm trying to include, but that might be easier to fix once I understand why we are using the wrong path to begin with.)

Reply:
Maybe we can sit together and debug this?